dockstore-client/src/main/java/io/dockstore/client/cli/YamlVerifyUtility.java

```diff
@@ -131,6 +136,9 @@ private static void allFilesExist(DockstoreYaml12 dockstoreYaml12, String basePa
             String errorMsg = errorMsgBuild.toString();
             throw new ValidateYamlException(MISSING_FILE_ERROR + errorMsg);
         }
+        if (!checkAbsoluteFilePaths(filePathsToCheck)) {
+            throw new ValidateYamlException(RELATIVE_PATH_ERROR);
+        }
```

Comment:
Realized this after - it might make more sense to move this absolute file check to line 121/126 before `missingFiles.addAll(filesExist(filePathsToCheck, basePath));`. We can avoid checking if files exist if the files have relative paths